src/index.js

```diff
@@ -1,9 +1,26 @@
 'use strict';
 
 const NewsEngine = require('./news-engine');
+const config = require('../config');
+const fs =  require(`fs`);
+const uuid = require(`uuid`);
+const svg2png = require('svg2png');
 
 const numExamples = process.argv[2] || 30;
 const headlines = NewsEngine.generateHeadlines(numExamples);
 headlines.forEach(headline => {
 	console.log("\n * " + headline.text);
+
+	if(headline.tags && headline.tags.svg && headline.tags.svg.svgString) {
+		const outputfile = uuid() + '.png';
+		const outputPath = config.paths.tempDirectory + '/' + outputfile;
+
+		svg2png(new Buffer(headline.tags.svg.svgString))
```

Comment:
Seems fine. We might want to put this behind an config flag that we can enable locally but can turn off for the bot, to keep the images from piling up.